aksel.nav.no/website/app/dev/(designsystemet)/_ui/sidebar/Sidebar.util.ts

```diff
@@ -98,6 +98,14 @@ function generateSidebar(
           });
         }
 
+        if (category.value === "styling") {
+          pages.unshift({
```

Comment:
Injecting this hard-coded page into the Sidebar. An optional method might be to create an empty page in Sanity with the same slug, a title and description, so we can do the "normal" Sanity fetch and use the normal header.